docker_publisher.py

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env python3
```

Comment:
please no /usr/bin/env :)

Reply:
Why not? The internet recommends that strongly as it takes `$PATH` into account and I'm not aware of any downsides

Reply:
because we package this repo and the require will be to /usr/bin/env - which is pointless. we want to require /usr/bin/python3. If you want python3 from your path, call it as python3 docker_publisher.py

Reply:
I thought the autoreq handler was intelligent enough to handle such cases? Apparently not.

Reply:
you're welcome to improve that. it would solve quite some problems :)

Reply:
in any case we're not some random ISV that repackages random python versions somewhere. This script will be for /usr/bin/python3 and nothing else.